transit_odp/timetables/dataframes.py

```diff
@@ -746,12 +746,23 @@ def populate_operating_profiles(
         )
         if days_of_operation:
             operational = True
-            serviced_orgs_working_days = days_of_operation.get_element("WorkingDays")
+            serviced_orgs_working_days = days_of_operation.get_element_or_none(
+                "WorkingDays"
+            )
+            if not serviced_orgs_working_days:
+                serviced_orgs_working_days = days_of_operation.get_element("Holidays")
+                if serviced_orgs_working_days:
```

Comment:
do we need to put same condition twice ... at line 752 also

if not serviced_orgs_working_days